pkg/apicapi/apic_metadata.go

```diff
@@ -154,8 +154,7 @@ func injectedSvcPortNormalizer(b *ApicObjectBody) {
 var metadata = map[string]*apicMeta{
 	"fvTenant": {
 		attributes: map[string]interface{}{
-			"name":      "",
-			"nameAlias": "",
+			"name": "",
```

Comment:
Was this change necessary?

Reply:
Yes. Since we are sending POST request to <tenant>.json and in the request body, if nameAlias is empty it was replacing the tenant alias (openupi) in our case to empty string